src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String stringSupply = "supply";
+    private static final String stringBuy = "buy";
+    private static final char ch = ',';
+    private static int intSupply = 0;
+    private static int intBuy = 0;
+    private static int result = 0;
```

Comment:
These variables should not be class variables, much less static. They should be local, try to fix it, the test should probably be successful.